docs/specs.md

```diff
@@ -25,6 +25,8 @@ We recommend that you always use the latest **stable** specification for your op
   - [v1.1.0](config-flatcar-v1_1.md)
   - [v1.0.0](config-flatcar-v1_0.md)
 - OpenShift (`openshift`)
+  - [v4.17.0](config-openshift-v4_17.md)
+  - [v4.16.0](config-openshift-v4_16.md)
```

Comment:
Ouch this was missing from the 4.16 PR. Good that we're fixing it here :)

Reply:
yeaaah